README.md

```
@@ -100,8 +100,8 @@ Join our growing community of users and contributors:
 
 ## Contributing
 
-<a href="https://github.com/theexperiencecompany/ui/graphs/contributors">
-  <img src="https://contrib.rocks/image?repo=theexperiencecompany/ui" />
+<a href="https://github.com/theexperiencecompany/gaia-ui/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=theexperiencecompany/gaia-ui" />
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)